index.js

```diff
@@ -69,6 +70,19 @@ module.exports = {
     return path.join(__dirname, 'blueprints');
   },
 
+  updateFastBootManifest(manifest) {
+    manifest.vendorFiles.push('ember-data/fetch-fastboot.js');
```

Comment:
I think we could lookup the ember-fetch addon and manually call `updateFastBootManifest` ourselves to be safer

Reply:
Ya, but this is slightly different. This PR is intentionally avoiding/working around the guard added by https://github.com/ember-cli/ember-fetch/pull/167 (which prevented the `treeForPublic` from returning anything when `ember-fetch` was not a top level addon).

Reply:
@rwjblue and I've also named this `ember-data/fetch-fastboot.js` instead of `ember-fetch/fetch-fastboot.js` like it would have been 🤔 I guess I could have done that but I was trying to avoid any name clashes. I don't know which way is better, I just went with my gut